package.json

```diff
@@ -17,7 +17,8 @@
     "package": "electron-packager . messenger-demo-viewer --out=dist --ignore='^/dist$' --prune --asar --all --icon=build/icon",
     "dist:win": "build -w --ia32 --x64",
     "dist:mac": "build -m",
-    "dist:lin": "build -l deb tar.gz --ia32 --x64"
+    "dist:lin": "build -l deb tar.gz --ia32 --x64",
```

Comment:
Can't I just do `"dist:lin": "build -l snap deb tar.gz --ia32 --x64",`?

Reply:
Yep, that should work as well.